CONTRIBUTING.md

```diff
@@ -0,0 +1,130 @@
+- The title of the Pull Request is used as proposal for the maintainer merging the Pull Request
+
+### Open a Pull Request
+
+At this point, you should switch back to your master branch and make sure it's up to date with `bpmn-visualization-examples`
+`master` branch:
+
+```sh
+git remote add upstream git@github.com:process-analytics/bpmn-visualization-examples.git
+git checkout master
+git pull upstream master
+```
+
+Then update your feature branch from your local copy of master, and push it!
+
+```sh
+git checkout 25-annotations_to_tasks
+git rebase master
+git push --set-upstream origin 25-annotations_to_tasks
+```
+
+Finally, go to GitHub and [make a Pull Request](https://help.github.com/articles/creating-a-pull-request) ​with labels :smile:
+For the title, follow the directives of the Pull Request template.
+Add a screenshot of the rendering of your examples/bpmn-file. 
+
+:warning: ​We care about quality. So your PR won't be merged until all tests pass.
+
+### Sign the Contributor License Agreement
+
+By signing the CLA, we acknowledge that your contribution is accepted as it and that you cannot be held responsible for
+any impacts on its integration. You then grant us the right to modify and distribute your code without restrictions. We
+ask this of all contributors in order to assure our users of the origin and continuing existence of the code.
+
+When you contribute to the project on GitHub with a new pull request, the [cla-assistant bot](https://cla-assistant.io/)
+will evaluate whether you have signed the CLA. If required, the bot will comment on the pull request, including a link
+to this system to accept the agreement.
+
+You only need to sign the CLA once or when the CLA terms have changed.
+
+### Keeping your Pull Request updated
+
+If a maintainer asks you to [rebase](http://git-scm.com/book/en/Git-Branching-Rebasing) your PR, they're saying that a lot of code has changed, and that you need to update your branch so it's easier to merge.
+
+Here's the suggested workflow:
+
+```sh
+git checkout 25-annotations_to_tasks
+git pull --rebase upstream master
+git push --force-with-lease 25-annotations_to_tasks
+```
+
+At this point, you're ready to make your changes! Feel free to ask for help. Everyone is a beginner at first :smile_cat:
+
+
+## Maintainers
+
+### Merging a PR (maintainers only)
+
+A PR can only be merged into master by a maintainer, if all of these conditions are met:
+
+* It is passing CI.
+* It has been approved by at least two maintainers. If it was a maintainer who opened the PR, only one extra approval is needed.
+* It has no requested changes.
+* It is up to date with current master.
+
+### Release (maintainers only)
+
+Maintainers need to do the manual following actions **in the order described here** to push out a release (the release
+automation is a work in progress).
+
+#### Git Tag
+
+The tag is done while the [bpmn-visualization-js](https://github.com/process-analytics/bpmn-visualization-js) release.
```

Comment:
:warning: to remove, currently describe in the lib repository and is expected to be automated in the future

Reply:
It's what I explain in this part, in order to avoid that someone tag in another moment 😉